SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -113,7 +127,7 @@ Interest InterestFactory::CreatePlayerOwnedActorInterest()
 
 	Interest NewInterest;
 	// Server Interest
-	if (DefinedConstraints.IsValid())
+	if (DefinedConstraints.IsValid() && GetDefault<USpatialGDKSettings>()->bEnableServerQBI)
```

Comment:
Is this redundant with the guard at line 44? (Fine to keep it, just making sure I understand.)

Reply:
This function `CreatePlayerOwnedActorInterest` gets called in both flows.